reconcile/test/fixtures/clusters/cluster1.yml

```diff
@@ -28,8 +28,8 @@ spec:
   private: false
   provision_shard_id: 'ashardid'
 network:
+  type: OpenShiftSDN
```

Comment:
Potentially a stupid question, but why we test with `OpenShiftSDN` instead of `OVNKubernetes`?

Reply:
I actually left the tests as is. They were relying on the default `OpenshiftSDN` being used, and were testing upgrades with the network type set to `OpenshiftSDN`. With the new default, this causes a test error stating the upgrade is not allowed.

Reply:
To avoid confusion, do you think it makes sense to add a comment there to highlight that this was the previous default?

Reply:
In the context of this PR, it makes sense to state it, and your comment makes so perfectly fine, thanks you!
In the context of the code & test sample, I don't really see what is changes whether that matches the current default value or not. So I would not add a comment in the code I think. Does that make sense?